<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectsForm</class>
 <widget class="QWidget" name="EffectsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    font-family : &quot;Open Sans&quot;;
    font-weight : 550;
    font-size:14pt;
}

#mainFrame{
    background-color : qlineargradient(spread:reflect, x1:0, y1:0.006, x2:0, y2:1, stop:0 rgba(223, 223, 223, 255), stop:1 rgba(159, 159, 159, 255));
}

QPushButton, #pbSave, #pbCancel, #pbSelect{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(149, 149, 149, 255));
border: 1px solid white;
color: black;
}

QPushButton:focus{ background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 114, 0, 255), stop:0.5 rgba(255, 137, 0, 255), stop:1 rgba(212, 114, 0, 255));}

QPushButton:focus:selected{ background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 114, 0, 255), stop:0.5 rgba(255, 137, 0, 255), stop:1 rgba(212, 114, 0, 255));}

#lbWidgetTitle{
    border: 1px solid black;
    background-color:rgb(16, 97, 114);
    color: white;
}

#lbCategoryName{
    border: 1px solid black;
    background-color:rgb(16, 97, 114);
    color: white;
    padding-left: 10px;
}

#lbCategory{
    border: 1px solid white;
    background-color:rgb(49, 51, 48);
    color: white;
}


/*------ QtableView ------*/
QTableView{
    border                  :   none;
    border-top              :   1px solid black;  
    border-left             :   1px solid black;   
    background-color        :   transparent;/*rgb(16, 97, 114);*/
    
}

QTableView::item{
    padding-left            :   20px;
    height                  :   75px;
    color                   :   white;  
    background-color        :   rgb(16, 97, 114);
    border                  :   1px solid black;
    border-top              :   1px solid transparent;
    /*border-right            :   1px solid transparent;*/
}

QTableView::item:selected{
    /*background-color        : rgb(244, 118, 75);*/
    background-color        :   qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(244, 131, 0, 255), stop:0.5 rgba(160, 86, 0, 255), stop:1 rgba(244, 131, 0, 255));

}


/*------ QHeaderView ------*/
QHeaderView{
    border                  :   none;
    background-color        :   rgb(65, 151, 48);
	font-size              : 14pt;
}

QHeaderView::section:vertical{
    margin-top              :   0px;
    margin-bottom           :   0px;
    border                  :   none;
    border-bottom           :   1px solid black;
    background-color        :   rgb(65, 151, 48);
    color                   :   white;
    width                   :   50px;
    font-weight             :   550;
    font-size                   : 14 pt;
}


/*------ QScrollbar ------*/
QScrollBar:vertical {
    border                  :   1px solid black;
    background              :   black;
    width                   :   35px;
    margin                  :   0 0 0 15px;
}

QScrollBar::handle:vertical {
    background              :   rgb(16, 97, 114);
    min-height              :   20px;
    margin: 1px 1px 1px 1px;
}


QScrollBar::add-line:vertical,QScrollBar::sub-line:vertical  {
    border                  :   none;
    background              :   transparent;
    height                  :   0px;
    margin                  :   0 0 0 10px;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    border                  :   none;
    width                   :   0px;
    height                  :   0px;
    background              :   white;
    margin                  :   0 0 0 10px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>480</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>14</pointsize>
       <weight>68</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="frameTop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>58</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>8</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="6">
          <widget class="QPushButton" name="pbEffactCat5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>EFFECT 
CAT5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lbWidgetTitle">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans</family>
               <pointsize>14</pointsize>
               <weight>68</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>EFFECTS</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pbEffactCat3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>EFFECT 
CAT3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pbEffactCat1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>EFFECT 
CAT1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="pbEffactCat4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>EFFECT 
CAT4</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pbEffactCat2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>55</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>EFFECT 
CAT2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>99</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbCancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>126</width>
            <height>58</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>126</width>
            <height>58</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>14</pointsize>
            <weight>68</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>CANCEL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>126</width>
            <height>58</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>126</width>
            <height>58</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>14</pointsize>
            <weight>68</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>SELECT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tvEffects">
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>14</pointsize>
          <weight>68</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>41</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
